CONTRIBUTING.md

```diff
@@ -101,68 +80,11 @@ Hi! This is my first post! (TODO: write more text here)
-
-## Embedded Links
-
-Blogs can embed their own `<iframe>` tags if necessary - these will initially show a "click to run" preview as to not affect the page's loading time.
-
-A handful of third-party services can also be embedded by simply pasting the link in the post content, including YouTube videos, Twitch clips, Twitter posts - and anything supported by [oembed.com](https://oembed.com).
-
-## Images & Videos
-
-If you have linked images or videos, you'll need to save those files in the same folder and change your markdown file to reference them locally:
-
-```markdown
-![Ferris, the adorable crustacean Rust mascot](./ferris.png)
-```
-
-> Make sure to add descriptive alt text! Consider what information these images add your post, and what context might be important for readers with visual impairment.
-
-Videos can also be embedded with the following syntax:
-
-```html
-<video src="./ios_vs_android.mp4" title="A comparison of how text spacing is applied on iOS and Android"></video>
-```
-
-> When possible, `<video>` elements should be preferred over `.gif` files or other animated images in our posts. This is for accessibility concerns - videos provide users with more control over when/how the animation plays.
-
-# Translating a Blog Post
-
-If you are adding a translation, make sure to first create an [Author Data File](#creating-an-author-profile) with the `"translator"` role so that you are credited for your work on the site!
-
-To create a translation file for a post, copy its `index.md` file and rename it to `index.(lang).md`, where `(lang)` is the translated language. For example, a translation for `fr` (French) would be named `index.fr.md`. The content inside this file can then be translated into the respective language.
-
-> If any images used in the post need to be translated, these should be named in a similar fashion - for example, a translation of `dom_tree.svg` should be named `dom_tree.fr.svg`.
->
-> Any links to these images will need to be updated in the `index.fr.md` post to point to the translated image.
-
-For reference, the current language codes can be found in [`/content/data/languages.json`](./content/data/languages.json) - you may need to add to this file if the language is missing.
-
-## Finding a Language Code
-
-Each language code in [`/content/data/languages.json`](./content/data/languages.json) should consist of two lowercase letters. If it includes a region, append a hyphen followed by two more lowercase letters. For example, the code for French is `fr` - to specifically refer to the French dialect in Canada, the code would be `fr-ca`.
-
-> Please use `-` instead of `_` in the language region ISO formats. Instead of `fr_ca`, it'd be `fr-ca`.
-
-Refer to [Wikipedia: List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for identifiers to be used in this format.
-
```

Comment:
We cannot remove this information to the CONTRIBUTING guide without relocating it somewhere else - just curious what was the thought process in doing so?